configs/setup_golden_image.py

```diff
@@ -41,17 +46,17 @@ def customize_golden(manconfig, golden_tar, build_dir):
         'golden_tar' : golden_tar,
         'build_dir' : build_dir,
         'status_file' : build_dir + '/status.json',
-        'verbose' : True,
+        'verbose' : VERBOSE,
```

Comment:
Also, if you want to stay with a Boolean, 'verbose': False  is much more legible than "verbose: VERBOSE"

Reply:
Not in this context. VERBOSE is set in main()